perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -193,11 +194,15 @@ def FillDevice(vm, disk, fill_size):
   vm.RemoteCommand(command)
 
 
-BENCHMARK_INFO = {'name': 'fio',
-                  'description': 'Runs fio in sequential, random, read '
-                                 'and write modes.',
-                  'scratch_disk': True,
-                  'num_machines': 1}
+BENCHMARK_NAME = 'fio'
+BENCHMARK_CONFIG = """
+fio:
+  description: Runs fio in sequential, random, read and write modes.
+  vm_groups:
+    default:
+      vm_spec: *default_single_core
+      disk_spec: *default_500_gb
```

Comment:
I was trying to keep the defaults the same - I agree that we should change this at some point, but I think that should be discussed and handled separately